src/components/Router/index.js

```diff
@@ -28,20 +28,23 @@ class CrossDeviceMobileRouter extends Component {
       token: null,
       steps: null,
       step: null,
-      socket: io(process.env.DESKTOP_SYNC_URL),
+      socket: io(process.env.DESKTOP_SYNC_URL, {autoConnect: false}),
```

Comment:
so if we don't have autoConnect how does the connection get re established?

Reply:
All `autoConnect` does is decouple creating a socket and opening it. Reconnection logic is handled elsewhere.

From the [docs](https://socket.io/docs/client-api/#manager):
`autoConnect` (Boolean) by setting this `false`, you have to call manager.open whenever you decide it's appropriate
`reconnection` (Boolean) whether to reconnect automatically (`true`)